src/components/structural/NavBar.tsx

```diff
@@ -1,42 +1,159 @@
 'use client'
```

Comment:
Design difference in figma
Current:
![image](https://github.com/Automatic-Survey-Generator/SurveyApp/assets/25122240/f3533660-f725-4944-a7c9-f3e9e1cee51c)
Original:
![image](https://github.com/Automatic-Survey-Generator/SurveyApp/assets/25122240/1cd1b702-906e-4b29-bf4b-55b040ad04bd)

- The header design is different to that of figma, which is fine - I like it, but we should discuss this with @LKPcode just in case. (I am not aware if you have talked about this)

Reply:
We haven't talked about it but i think it looks good as well.
The thing is that the Navbar links will not stay (reason in bottom comment) and the corner might look empty, we ll see :)